drivers/ethernet/Kconfig.dsa

```diff
@@ -4,6 +4,15 @@
 #               Lukasz Majewski <lukma@denx.de>
 # SPDX-License-Identifier: Apache-2.0
 
+menuconfig NET_DSA
+	bool "Distributed Switch Architecture support"
+	depends on NETWORKING
```

Comment:
I see your point. Thanks for spotting this.
Indeed - IMHO, it would be better if DSA would depend on ETH_MCUX (as for now we only work with this driver).

If other architecture would like to add it - then one can add:
`       depends on ETH_MCUX || ETH_FOO`